pay-api/src/pay_api/services/refund.py

```diff
@@ -137,7 +137,7 @@ def create_refund(cls, invoice_id: int, request: Dict[str, str], **kwargs) -> No
 
         refund: RefundService = RefundService()
         refund.invoice_id = invoice_id
-        refund.reason = request.get('refund', None)
+        refund.reason = request.get('reason', None) if request else None
```

Comment:
I use getAttr for this usage.. getAttr(request,reason,None) ..not sure which is better

Reply:
looks like getattr doesn't work  for dict?  I have a dpath implementation for this, can use that one.

Reply:
Pushed change